openlibrary/solr/update_work.py

```diff
@@ -1452,7 +1452,7 @@ def solr_escape(query):
     :param str query:
     :rtype: str
     """
-    return re.sub('([\s\-+!()|&{}\[\]^\"~*?:\\\\])', r'\\\1', query)
+    return re.sub(r'([\s\-+!()|&{}\[\]^"~*?:\\])', r'\\\1', query)
```

Comment:
Some of the remaining escapes (e.g. `\[`) aren't strictly necessary, but they also don't seem to cause any problems. I'm not convinced that the `\s` is correct, but that's unrelated to the update.